backend/HoReD/Controllers/RegistrationController.cs

```diff
@@ -64,10 +64,11 @@ public IHttpActionResult ActivateUser(string IdUser)
                int result=_userService.ActivateUser(decryptedUserId);
                 return Ok(result);
             }   
-            catch (Exception )
+            catch (Exception e )
             {
-             
-                return Unauthorized();
```

Comment:
Why removed?
Also, need to remove Console.WriteLine() and throw

Reply:
I need it for debug in Azure